scene/2d/collision_shape_2d.cpp

```diff
@@ -110,6 +110,7 @@ void CollisionShape2D::_notification(int p_what) {
 				draw_col.r = g;
 				draw_col.g = g;
 				draw_col.b = g;
+				draw_col.a /= 2;
```

Comment:
Probably a tiny optimization, but since it can be called every frame, it might be worth doing it:

```suggestion
				draw_col.a *= 0.5;
```